nikola/plugins/compile/html.py

```diff
@@ -49,7 +48,7 @@ def compile_string(self, data, source_path=None, is_two_file=True, post=None, la
         if not is_two_file:
             _, data = self.split_metadata(data)
         new_data, shortcodes = sc.extract_shortcodes(data)
-        return self.site.apply_shortcodes_uuid(new_data, shortcodes, filename=source_path, with_dependencies=True, extra_context=dict(post=post))
+        return self.site.apply_shortcodes_uuid(new_data, shortcodes, filename=source_path, extra_context={'post': post})
```

Comment:
Ah, I didn't notice that you changed the API in `nikola.py` since apparently github was hiding its diff and so I couldn't find it. I now saw it when checking out the branch and looking in there...